_packages/native-preview/lib/getExePath.js

```diff
@@ -0,0 +1,43 @@
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+
+export default function getExePath() {
+    const __dirname = path.dirname(fileURLToPath(import.meta.url));
+    const normalizedDirname = __dirname.replace(/\\/g, "/");
+
+    let exeDir;
+
+    const expectedPackage = "native-preview-" + process.platform + "-" + process.arch;
+
+    if (normalizedDirname.endsWith("/_packages/native-preview/lib")) {

```

Comment:
dirname never has a trailing slash, right?

Reply:
It doesn't, no.